sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -96,6 +96,42 @@ static void str_toupper(char *str)
     }
 }
 
+__attribute__ ((unused))
+static uint8_t gcd(uint8_t a, uint8_t b)
```

Comment:
there is one in `frac.c` can't you use that?

Reply:
yea that just looked so complex and I didn't need speed here 😅

Reply:
You lost me a bit a here, where would this database/table be used?

Reply:
As with WiFi different world regions have varying regulations as to what portions of the spectrum you may use and how you may use it. And while the 2.4 GHz ISM band can be used everywhere, for sub-GHz different countries use different frequencies for ISM and the [standard](https://sci-hub.se/https://ieeexplore.ieee.org/document/9144691) prescribes different modulation parameters for those frequencies.
 
![image](https://user-images.githubusercontent.com/1301112/96842584-8321cd80-144d-11eb-9059-6143d7760c8d.png)

![image](https://user-images.githubusercontent.com/1301112/96842703-a6e51380-144d-11eb-9868-243cff16b745.png)

So in the end you would really want to be able to "set region EU, set fsk_mode 3" and not worry about the details.

But that should be handled outside the driver so other ~~MR~~SUN-FSK like `cc26x2_cc13x2_rf` can also benefit from it.